src/Microsoft.DocAsCode.Build.Common/Reference/ApplyOverwriteDocument.cs

```diff
@@ -92,6 +92,8 @@ select new
                 Merge(vm, ovm, ovms[0].fileModel);
                 pair.model.LinkToUids = pair.model.LinkToUids.Union(od[0].LinkToUids);
                 pair.model.LinkToFiles = pair.model.LinkToFiles.Union(od[0].LinkToFiles);
+                pair.model.FileLinkSources = pair.model.FileLinkSources.Union(od[0].FileLinkSources).ToImmutableDictionary();
```

Comment:
I'll fix it in next separate review, to distinguash with this one.

---
In reply to: [99538801](https://github.com/dotnet/docfx/pull/1269#discussion_r99538801) [](ancestors = 99538801,99538362)